### PR TITLE
Liquid use case edits pt.2

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md [hunk 1]
```diff
@@ -1078,13 +1079,6 @@ This use case checks if a custom attribute array contains a specific string, and
 ```liquid
 {% if custom_attribute.${PartnershipProgramsNotLinked} contains 'Hertz' %}
 Link your Hertz account to use Hertz Fast Lane.
-
-{% elsif custom_attribute.${airportCompleted} == false %}
-Clear helps you breeze through airport security. Complete your one-time in-person setup next time you are at the airport. It only takes about 5 minutes.
-
-{% else %}
-Your account is all setup
-{% endif %}
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:1087]: For my understanding, what is incorrect/inaccurate about this section of the Liquid? I believe deleting the `endif` statement will break this.

Comment (rachel-feinberg) [reply _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:1087]: The use case tester suggested to remove that portion as it's not specific to the example (finding a string in an array)

It’s fine but I would include this part in the example (the other parts don’t really apply to the example at hand). Can just close the condition statement after that if statement: 
{% if custom_attribute.${PartnershipProgramsNotLinked} contains 'Hertz' %}
Link your Hertz account to use Hertz Fast Lane.

Comment (bre-fitzgerald) [reply _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:1087]: Ah okay, I believe you'll want to add the `{% endif %}` statement back in then, as that would be closing the condition statement.

Comment (bre-fitzgerald) [reply _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md]: It's like in the alerts we use in docs (which also use Liquid), if you add `{% alert note %}` but don't include `{% endalert %}`, the whole thing goes 💥

Comment (rachel-feinberg) [reply _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md]: Ahhh that's for catching that

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**